docker-compose.yaml

```diff
@@ -31,5 +33,19 @@ services:
       - POSTGRES_USER=lugo
       - POSTGRES_PASSWORD=passw0rd
       - POSTGRES_DB=lugo
+  redis:
+    container_name: redis
+    image: redis:7-alpine
+    ports:
+      - '6379:6379'
+    healthcheck:
+      test: ['CMD-SHELL', 'redis-cli ping | grep PONG']
```

Comment:
`| grep PONG` ensures the command will succeed only if the db is really ready. [Reference](https://stackoverflow.com/a/71504657/7178837).